styles.css

```diff
@@ -140,7 +135,317 @@ header a {
-.headline-social li img {
+.project {
+  display: grid;
+  grid-template-columns: 375px;
+  align-items: center;
+  margin-top: 718px;
+  background-color: #e5e5e5;
+  height: auto;
+  width: 375px;
+  gap: 88px;
+  padding-top: 114px;
+}
+
+.project .card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  width: 327px;
+  min-height: auto;
+  max-height: 596px;
+  border-radius: 16px;
+  margin-left: 24px;
+  padding-bottom: 20px;
+}
+
+.project img {
+  height: 220px;
+  width: 295px;
+  border-radius: 8px;
+  margin: 16px;
+}
+
+.headline ul img {
   height: 20px;
   width: 20px;
 }
+
+.card .small-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 295px;
+  height: 36px;
+  margin: 0;
+}
+
+.card .long-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 295px;
+  height: 88px;
+  margin: 0;
+}
+
+.card .info {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 250px;
+  height: 20px;
+  list-style: none;
+  margin-top: 20px;
+  margin-left: -85px;
+  display: flex;
+  justify-content: flex-start;
+  gap: 12px;
+}
+
+.about-language li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 10px;
+  margin-top: 10px;
+}
+
+.card .info li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+  display: flex;
+  align-self: center;
+}
+
+ul .fa-solid {
+  color: #c1c7d0;
+  font-size: 8px;
+}
+
+.card .info li:first-child {
+  font-weight: 600;
+  color: #344563;
+}
+
+.card p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  width: 295px;
+  height: 76px;
+  margin-top: 0;
+}
+
+.card .program {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  width: 200px;
+  height: 32px;
+  margin: 0 0 0 -135px;
+  display: flex;
+  list-style: none;
+  gap: 8px;
+}
+
+.about-language .language {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  width: 280px;
+  height: 60px;
+  margin-left: -33px;
+  list-style: none;
+  background-color: #f7f7f9;
+  border-radius: 10px;
+  margin-top: 12px;
+  padding-left: 10px;
+  color: #253858;
+  gap: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.card .program .language {
+  font-size: 12px;
+  height: 24px;
+  width: auto;
+  padding: 4px 12px 4px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  background-color: #ebebff;
+  display: flex;
+  align-self: center;
+}
```

Comment:
***[OPTIONAL]*** 

- [ ] In [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=99Sj0XRcegz7sbHC-0) inside the card `html`, `css`, `javascript`, etc elements are centered inside the `button` or `li`. Kindly fix that.

- `Your Design`
![Screenshot from 2022-12-03 16-25-57](https://user-images.githubusercontent.com/60083437/205437097-8d631cf0-eb84-41d5-96cc-92aa4caed3b9.png)

- `Given Figma Design`
![hello](https://user-images.githubusercontent.com/60083437/205437232-a4f28b46-2119-4561-9edb-f9a99473f2d1.jpg)